docs/software/media.md

```diff
@@ -359,49 +359,35 @@ Remark: If you do not own a hardware player, you can turn your DietPi system int
 
     - `/mnt/dietpi_userdata/Music`, `/Music` from NFS/Samba
 
-=== "Change Squeezelite command line options"
+=== "View logs"
 
-    - Run `dietpi-services`
-    - Select `squeezelite`
-    - Select `Edit`
-    - Unset and re-set the `ExecStart` entry:
+    Logs can be viewed with the following command:
 
-      ```systemd
-      ExecStart=
-      ExecStart=/usr/bin/squeezelite [<your custom arguments>]
-      ```
+    ```sh
+    journalctl -u logitechmediaserver
+    ```
 
-      The first `ExecStart=` is required to replace the existing `ExecStart` entry instead of adding a second one.
+    Additionally, log files can be found in:
 
-    - Save changes with ++ctrl+o++ and exit `dietpi-services`
-    - Restart the service: `systemctl restart squeezelite`
+    ```
+    /var/log/squeezeboxserver
+    ```
 
 === "Update"
 
-    To update Logitech Media Server to the current version, execute
+    Logitech Media Server can be updated to the current version with the following command:
 
     ```sh
     dietpi-software reinstall 35
     ```
 
-## Squeezelite
-
-Squeezelite is the audio player for the Logitech Media Server.
-
-![Logitech Media Server web interface screenshot](../assets/images/dietpi-software-media-squeezebox.png){: width="400" height="216" loading="lazy"}
-
-=== "Access to the web interface"
-
-    The web interface is accessible via port **9000**:
+***
 
-    - URL: `http://<your.IP>:9000`
+Source code: <https://github.com/Logitech/slimserver>
 
-=== "Transfer music to DietPi"
-
-    Make sure you have one of DietPi's [File Servers](../file_servers/) installed.  
-    Default music directory:
+## Squeezelite
 
-    - `/mnt/dietpi_userdata/Music`, `/Music` from NFS/Samba
+Squeezelite is the audio player for the Logitech Media Server.
```

Comment:
Proposal: Squeezelite is a command line based audio player for the Logitech Media Server (headless).

Reply:
> Squeezelite is a command line based audio player

It is not really a CLI audio player, i.e. one cannot use it to play a local audio file. It is really just the backend for LMS: https://en.wikipedia.org/wiki/Squeezelite

Probably we should use "client" or "backend" here, instead of "player"?